openverse_catalog/dags/common/loader/reporting.py

```diff
@@ -38,11 +38,18 @@ def report_completion(provider_name, media_type, duration, record_count):
     if isinstance(duration, float):
         duration = humanize_time_duration(duration)
 
+    # List record count per media type
+    media_type_reports = "\n".join(
+        f"  - `{media_type}`: {record_count}"
```

Comment:
It looks like when 0 records are pulled, the `load_data` task is skipped so we get `None` for `record_count` here. Although I guess it is possible to pull some records but upsert 0, if they're all removed during the [cleaning step](https://github.com/WordPress/openverse-catalog/blob/38ee4938afa7794ff56e92cb162ce0348838ef33/openverse_catalog/dags/common/loader/sql.py#L159) 🤔 

I wish we could get the status of the individual `load_data` tasks here, so we could distinguish between 0 pulled and an error.

Reply:
Oh! That's good to know, I guess that `None` makes sense there then.

We could get the `load_data` state by querying the task instance records, but I think that'd be a pain and more effort than it's worth in this case :confused:

Reply:
That's what I thought too -- it would be rad if there was a simpler way through XCom, in the same way that the task instance's `return_value` is automatically communicated 😞 That would be useful for so many things!